pkgs/by-name/no/nomacs/package.nix

```diff
@@ -8,6 +8,9 @@
 , opencv4
 , pkg-config
 , stdenv
+, config
+, cudaSupport ? config.cudaSupport
+, cudatoolkit
```

Comment:
https://github.com/NixOS/nixpkgs/pull/260745#discussion_r1357295158

Don't use cudatoolkit

Reply:
Done, I have switched to cudaPackages. Could you please check my PR again?